src/GameLogic/Enums.ts

```diff
@@ -98,4 +98,8 @@ export class PlayerActionUtils {
 
 		return direction;
 	}
+
+	public static isMoveAction(action: PlayerAction): boolean {
+		return action >= PlayerAction.MoveUpLeft && action <= PlayerAction.MoveDownRight && action !== PlayerAction.Wait;
```

Comment:
Are the range checks really necessary? Does typescript not enforce that this enum stay within bounds anyway?

Reply:
Right now it makes little sense indeed, but in the future we'll likely need more actions - I think we had a conversation where we agreed we don't want to allow recording moves into walls, which later are replayed as actual walk actions rather than bumps into the wall. Once/if we decide to do it like that this'll starting having value.